src/components/UI/Tag.tsx

```diff
@@ -0,0 +1,30 @@
+import { CloseCircleOutline, CloseCircle } from 'react-ionicons';
+import { twMerge } from 'tailwind-merge';
+import { useState } from 'react';
+
+const Tag = ({ text, styles }: { text: string; styles?: string }) => {
+  const [isHoaver, setIsHoaver] = useState(false);
+
+  return (
+    <div>
+      <span
+        className={twMerge(
+          `bg-green-500 text-white flex gap-1 items-center px-2 rounded-sm cursor-pointer fill-white text-sm font-semibold ${
+            styles ?? ''
+          }`,
+        )}
+        onMouseEnter={() => setIsHoaver(true)}
```

Comment:
"Hoaver" is not a work, please use setIsHover, and so on...

Reply:
Por favor asegurate que se vea como en el diseño, veo muy descuidado el componente, se corre a todo el ancho de la pantalla los iconos cambian de tamano, no tiene bordes redondeados, le falta padding....